pkg/driver/sanity_test.go

```diff
@@ -18,20 +18,25 @@ import (
 	"sigs.k8s.io/ibm-powervs-block-csi-driver/pkg/util"
 )
 
+var (
+	// TargetPath ...
+	TargetPath = ""
+)
+
 func TestSanity(t *testing.T) {
 	// Setup the full driver and its environment
-	dir, err := ioutil.TempDir("", "sanity-ebs-csi")
+	dir, err := ioutil.TempDir("", "sanity-powervs-csi")
 	if err != nil {
 		t.Fatalf("error creating directory %v", err)
 	}
 	defer os.RemoveAll(dir)
 
-	targetPath := filepath.Join(dir, "mount")
+	TargetPath = filepath.Join(dir, "mount")
 	stagingPath := filepath.Join(dir, "staging")
 	endpoint := "unix://" + filepath.Join(dir, "csi.sock")
 
 	config := &sanity.Config{
-		TargetPath:       targetPath,
```

Comment:
I'm wondering whether we can just pass this `targetPath` into the `MockStatSanity` struct and use it instead of making this global?

Reply:
Modified as suggested.